docs/manual/src/docs/asciidoc/_includes/reactive/webflux.adoc

```diff
@@ -124,3 +124,66 @@ class HelloWebfluxSecurityConfig {
 
 This configuration explicitly sets up all the same things as our minimal configuration.
 From here you can easily make the changes to the defaults.
+
+You can find more examples of explicit configuration in unit tests, by searching https://github.com/spring-projects/spring-security/search?q=path%3Aconfig%2Fsrc%2Ftest%2F+EnableWebFluxSecurity[EnableWebFluxSecurity in the `config/src/test/` directory], e.g. https://github.com/spring-projects/spring-security/blob/9cf3129d7afa2abb439aba6aadfee0a2c8c784bf/config/src/test/java/org/springframework/security/config/annotation/web/reactive/EnableWebFluxSecurityTests.java#L349-L366[MultiSecurityHttpConfig] illustrating multiple `SecurityWebFilterChain` beans, which is also explained in section below.
+
+=== Multiple chains support
+
+We can configure multiple `SecurityWebFilterChain` instances.
+
+For example, the following is an example of having a different configuration for URL's that start with `/v2/`
+
+[source,java]
+----
+@Configuration
+@EnableWebFluxSecurity
+public class MultiWebfluxHttpSecurityConfig {
+
+	@Bean
+	public MapReactiveUserDetailsService userDetailsService() {
+		UserDetails user = User.withDefaultPasswordEncoder()
+			.username("user")
+			.password("user")
+			.roles("USER")
+			.build();
+		return new MapReactiveUserDetailsService(user);
+	}
+
+
+	@Order(SecurityProperties.BASIC_AUTH_ORDER - 10)                                                     <1>
+	@Bean
+	public SecurityWebFilterChain osbUnRestrictedSpringSecurityFilterChain(ServerHttpSecurity http) {
```

Comment:
I'd recommend simplifying the example as much as possible, so as to keep the focus on the principle you are illustrating.

Consider starting with `MultiSecurityHttpConfig` from `EnableWebFluxSecurityTests` and making it more complex, if you see the need.

Reply:
In this specific example, the `MultiSecurityHttpConfig` seems to rely on defaults for its second `SecurityWebFilterChain`. 

https://github.com/spring-projects/spring-security/blob/9cf3129d7afa2abb439aba6aadfee0a2c8c784bf/config/src/test/java/org/springframework/security/config/annotation/web/reactive/EnableWebFluxSecurityTests.java#L349-L366

and it was not clear to me how readers would discover those defaults that lead to others requests such as `/test`be accepted.
https://github.com/spring-projects/spring-security/blob/9cf3129d7afa2abb439aba6aadfee0a2c8c784bf/config/src/test/java/org/springframework/security/config/annotation/web/reactive/EnableWebFluxSecurityTests.java#L247-L264

To match the existing servlet api example at https://github.com/spring-projects/spring-security/blob/1ca47f5cd0c5ca574f39d25bed82f8b83ced7732/docs/manual/src/docs/asciidoc/_includes/servlet/java-configuration/index.adoc#multiple-httpsecurity I've therefore reused an explicit distinct config inspired from `WebfluxFormSecurityConfig`.